tests/src/helpers.c

```diff
@@ -282,3 +282,32 @@ void mbedtls_param_failed( const char *failure_condition,
     }
 }
 #endif /* MBEDTLS_CHECK_PARAMS */
+
+#if defined(MBEDTLS_TEST_HOOKS)
+void mbedtls_test_err_add_check( int high, int low,
+                                 const char *file, int line )
+{
+    if ( high > -0x1000 )
```

Comment:
I think that apart from `low > 0`, the checks are complete and correct. However:

* They would be easier to read with comparisons of positive numbers, and for some of them, with bit masks. A {high,low}-level error code is a value whose opposite has only the bits {7–14,0–6} set. We don't currrently use high-level error codes whose upper 3 bits are 0, but `error.h` still defines them as high-level error codes.

Reply:
Done.